src/admin/settings/settings.html

```diff
@@ -0,0 +1,126 @@
+<div ng-if="!platformSettings.readOnly" class="form-group settings-box-medium">
+    <div class="settings-box-content">
```

Comment:
Lets junk the settings-box-content classes (and divs) -- they add almost no value. I recommend turning them into forms or fieldsets within a larger form...